entrypoints/internal.d.ts

```diff
@@ -0,0 +1,25 @@
+import type {StateChangeEvent} from '../types/state-change-events.d.cts';
+
+export type RunEvent = {
+	type: 'stateChange';
+	stateChange: StateChangeEvent;
+} | {
+	type: 'run';
+	plan: {
+		bailWithoutReporting: boolean;
+		debug: boolean;
+		failFastEnabled: boolean;
+		filePathPrefix: string;
+		files: string[];
+		matching: boolean;
+		previousFailures: number;
+		runOnlyExclusive: boolean;
+		firstRun: boolean;
+	};
```

Comment:
Do you think this is useful? I wonder if we should just emit the state change events? And name those `Event`.

Reply:
f10831b5